ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -108,7 +108,8 @@ def initialize(self: Analysis) -> None:
         FileHandler(jedi_fix_list).sync()
 
         # stage backgrounds
-        FileHandler(self.get_bkg_dict()).sync()
+        logger.debug(f"Stage ensemble member background files")
+        FileHandler(Analysis.get_ens_dict(self, self.task_config)).sync()
```

Comment:
```suggestion
        FileHandler(self.get_ens_dict(self.task_config)).sync()
```
`atmens_analysis.py` inherits from `analysis.py`.

Reply:
Accept suggestion.  Done at 5e88371.